test/core/modules/image-sequencer.js

```diff
@@ -103,29 +104,54 @@ test('addSteps("name",o) adds a step', function(t) {
   t.end();
 });
 
+test('addSteps("name,name") adds two steps', function(t) {
+  sequencer.addSteps('channel,invert')
+  t.equal(sequencer.steps.length, 7, 'Length of steps increase')
+  t.equal(sequencer.steps[5].options.name, "channel", "Correct Step Added");
+  t.equal(sequencer.steps[6].options.name, "invert", "Correct Step Added");
+  t.end()
+})
+
+test('addSteps("name{parameter:value},name{parameter:value}") adds two steps', function(t) {
+  sequencer.addSteps('brightness{brightness:80},average{}')
+  t.equal(sequencer.steps.length, 9, 'Length of steps increase')
```

Comment:
Isn't there a better way to keep track of how many steps were added?

Reply:
We're brute forcing our tests anyway. We could start removing steps after each test suite, but I think it's fine for now. Maybe a future issue when tests are being overhauled